docs/admin-connections/0.1/README.md

```diff
@@ -6,14 +6,19 @@ Connections Admin Protocol
 **Protocol URI:** `https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-connections/0.1`
 
 ### Protocol Messages
-- [connection](#connection)
-- [get-list](#get-list)
-- [list](#list)
-- [update](#update)
-- [delete](#delete)
-- [deleted](#deleted)
-- [receive-invitation](#receive-invitation)
-- [connected](#connected)
+- [Connections Admin Protocol](#connections-admin-protocol)
+  - [Overview](#overview)
+    - [Protocol Messages](#protocol-messages)
+  - [Message Definitions](#message-definitions)
+    - [Connection](#connection)
+    - [Get List](#get-list)
+    - [List](#list)
+    - [Update](#update)
+    - [Delete](#delete)
+    - [Deleted](#deleted)
+    - [Receive Invitation](#receive-invitation)
+    - [Receive OOB Invitation](#receive-oob-invitation)
+    - [Connected](#connected)
```

Comment:
While your changes are better formatted, the original formatting was actually ugly for a specific reason (this time at least lol): some scripts in this repo read from the `Protocol Messages` list and generate folders for each message type. I recommend we keep the formatting as it was originally (plus the new OOB message).